Azure/sonic-buildimage
# [Nokia] Update Nokia sonic-platform submodule

#### Why I did it
Update nokia sonic-platform submodule with following commits.  The latest submodule includes changes of the PMON reboot API on Supervisor card supports reboot of all its linecards. To fully support all new platform features on Nokia platform, this PR may requires the latest NDK.

```
  1de25f6 Modified module.py reboot() to reboot all IMM when reboot API is issued on CPM.
  ca45505 PMON SFP: mods so that test code can make API calls without first requiring get_transceiver_info() and get_transceiver_bulk_status() calls
  c16904b Added nokia_cmd to start/stop nokia-xcvr-resync.service
  c1d27a4 PMON SFP API: fix tx_disable_channel and power_override support
  6871167 Merge pull request #4 from nokia/sakthi_sfp_reset
  da5b5fc Addressed code review comments
  0ab6be9 Changed the sfp reset design to reset when the admin is up and oper is down
  ccbff9f Add support for SFP reset
  81b1410 Changed the asic thermal temperature name to capital letter ASIC instead of asic
  6669936 Added midplane link-status-flap command to show the ethmgr port flaps
  9a7ca27 Modified Qfpga nokia_cmd cli commands to use meaningful qfpga port names.
  bd58726 Added CLI commands to show QFPGA port status, port statistics, error-counters, epipe counters
  51a6559 Fixed situation where transceiver module status is incorrect after dynamic removal/insertion 
  487d4ed Fixed the syntax error on nokia_install_ndk_service
  1772002 Fixed code review comments and some code cleanup for Asic Thermal
  25af678 Added support for Asic Thermal
  c24b16a Add nokia_cmd cli support co clear midplane port counters
```
#### How I did it
Update the Nokia sonic-platform submodule. 

#### How to verify it
On Supervisor card, do the following to use the PMON reboot() API to reboot the supervisor card.  And to verify that all Linecards  should be reboot also.
```
root@supervisor:/# python3
Python 3.9.2 (default, Feb 28 2021, 17:03:44) 
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sonic_platform
>>> sonic_platform.platform.Platform().get_chassis()._get_my_module().reboot("default")
```

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
